### PR TITLE
Display a score out of 100

### PR DESCRIPTION
**Pull request by magopian**
cf [notion](https://www.notion.so/fd0068a7db8348c584cd53a3eea399f2?v=90c1a9d44ff845c090b6f491065ba2ef&p=c0d9599591084dbeba62132aa8c49202&pm=s)

### CODE REVIEW

File: src/Views/Format.elm [hunk 1]
```diff
@@ -63,6 +64,48 @@ formatFoodSelectedImpactPerKg { trigram, unit, decimals } totalMass =
         >> formatRichFloat decimals (unit ++ "/kg")
 
 
+formatFoodSelectedImpactScore : Impact.Definition -> Mass -> Impacts -> Html msg
+formatFoodSelectedImpactScore { trigram } totalMass impacts =
```
Comment (n1k0) [inline src/Views/Format.elm:68]: Let's suppose for a minute we want to expose this score to the API; would we be happy with an HTML only function? Even if that's not a requirement atm, wouldn't it be nice to have the computation extracted to a more agnostic function retyurning an `Int`, with maybe a few unit tests added?

File: src/Views/Format.elm [hunk 2]
```diff
@@ -63,6 +64,48 @@ formatFoodSelectedImpactPerKg { trigram, unit, decimals } totalMass =
...
+                "B"
+
+            else if score >= 40 then
+                "C"
+
+            else if score >= 20 then
+                "D"
+
+            else
+                "E"
```
Comment (n1k0) [inline src/Views/Format.elm:99]: I couldn't find a requirement for rendering a letter, is that an informal one?

**Review by n1k0 — COMMENTED** [review]
Nice :+1: just a few comments

File: src/Views/Format.elm (hunk 2 above)
Comment (magopian) [reply src/Views/Format.elm:99]: yep, a discussion with Lisa about adding this as an experiment and base for future discussions

File: src/Views/Format.elm [hunk 3]
```diff
@@ -63,6 +64,53 @@ formatFoodSelectedImpactPerKg { trigram, unit, decimals } totalMass =
...
+            else if score >= 60 then
+                "B"
+
+            else if score >= 40 then
+                "C"
+
+            else if score >= 20 then
+                "D"
+
+            else
```
Comment (n1k0) [inline src/Views/Format.elm]: We don't need any of this now it's been moved to Impact.elm

File: src/Data/Impact.elm [hunk 4]
```diff
@@ -403,6 +405,42 @@ getAggregatedScoreData defs getter =
...
+            logBase e
+    in
+    impacts
+        |> perKg totalMass
+        |> getImpact trigram
+        |> Unit.impactToFloat
+        |> (\value ->
+                -- See the documentation at https://fabrique-numerique.gitbook.io/ecobalyse/alimentaire/impacts-consideres/score-100
+                (ln 2077 - ln value) / ln 2 * 20
+           )
```
Comment (n1k0) [inline src/Data/Impact.elm:421]: I'd really like to see at least a soingle unit test added, just for the sake of non-regression control over time

**Review by n1k0 — APPROVED** [review]
Perfect, ship it :rocket:

**Pull request merged**